build/migration/1626228405913-CreateUserTable.js.map

```diff
@@ -1 +1 @@
-{"version":3,"file":"1626228405913-CreateUserTable.js","sourceRoot":"","sources":["../../migration/1626228405913-CreateUserTable.ts"],"names":[],"mappings":";;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;AAEA;IAAA;IAmBA,CAAC;IAjBgB,yCAAE,GAAf,UAAgB,WAAwB;;;;4BACpC,qBAAM,WAAW,CAAC,KAAK,CACnB,gTAM2B,CAAC,EAAA;;wBAPhC,SAOgC,CAAC;;;;;KACpC;IAIY,2CAAI,GAAjB,UAAkB,WAAwB;;;;4BACtC,qBAAM,WAAW,CAAC,KAAK,CACnB,6BAA6B,CAAC,EAAA;;wBADlC,SACkC,CAAC;;;;;KACtC;IACL,mCAAC;AAAD,CAAC,AAnBD,IAmBC;AAnBY,oEAA4B"}
```

Comment:
We don't need to change on `build` folder. Because this folder deleted and rebuild to deploy.
You can create new migrate file to change your database. https://typeorm.io/#/migrations

Reply:
Sorry for having done it.